pkg/cli/admin/upgrade/upgrade.go

```diff
@@ -79,7 +79,7 @@ func New(f kcmdutil.Factory, parentName string, streams genericclioptions.IOStre
 	flags.StringVar(&o.ToImage, "to-image", o.ToImage, "Provide a release image to upgrade to. WARNING: This option does not check for upgrade compatibility and may break your cluster.")
 	flags.BoolVar(&o.ToLatestAvailable, "to-latest", o.ToLatestAvailable, "Use the next available version")
 	flags.BoolVar(&o.Clear, "clear", o.Clear, "If an upgrade has been requested but not yet downloaded, cancel the update. This has no effect once the update has started.")
-	flags.BoolVar(&o.Force, "force", o.Force, "Upgrade even if an upgrade is in process or other error is blocking update.")
+	flags.BoolVar(&o.Force, "force", o.Force, "Try forcully upgrade the cluster but check for upgrade possibility.")
```

Comment:
Suggested wording:  Forcefully upgrade the cluster even when upgrade release image validation fails.

Reply:
Fixed.